plugins/destination/postgresql/client/migrate.go

```diff
@@ -189,10 +187,10 @@ func (c *Client) addColumn(ctx context.Context, tableName string, column schema.
 
 func (c *Client) createTableIfNotExist(ctx context.Context, table *schema.Table) error {
 	var sb strings.Builder
-	tName := table.Name
-	tableName := pgx.Identifier{tName}.Sanitize()
+	tableName := table.Name
```

Comment:
Renamed these since I initially used `tableName` for the dictionary key which is wrong since it had the sanitized value